packages/api/types.d.ts

```diff
@@ -216,6 +216,7 @@ declare global {
 	type GithubIndicator = {}
 	type BlueAsset = {
 		id: number
+		// hash: string
```

Comment:
can just write `hash?: string` and also remove the comment in `DataBuilder`

Reply:
not sure why, don't think I'm going to forget how to hash etc, I hope.

Reply:
I meant the comment in the blueasset block

Reply:
https://github.com/VandeurenGlenn/blue_server/pull/76/files#diff-ac184c21e1621a79bd8c279b7260877562aa6b8b90588a466a96810aae0c1d37R56

Reply:
Why? we don't support hash now, so just keep all commented ? 
Or just remove all hash related for now till we want it
but adding it in the type as optional when not going to use hash makes no sense?

Reply:
when you write `hash?: string` in the type but do not include in the target object it just doesn't exist.